week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,47 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// The Math. random() function is used to return a floating-point pseudo-random number between range [0,1)
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
```

Comment:
You explained what .random() does, but you had been asked to explain what the getNumber function does. You should give an explanation of it.